doc/Programming-guide/Key-platform-features/Endpoint-profiles/index.md

```diff
@@ -87,42 +98,41 @@ The following client-side profile is based on the rules that sets schema.
   "build" : "1"
 }
 ```
-The schema structure from our example allows filtering the endpoints by the operation systems of device (for example, to show only Android devices), os_version (for example, to push some 
-notifications only for the specified os version). It is allowed to create complex filtering conditions by combining as many filtering conditions as needed.
 
-### Setting client-side endpoint profile schema from Admin UI
+The schema structure from our example allows filtering the endpoints by the operation system of device (for example, to show only Android devices), os_version (for example, to push some notifications only for the specified os version).
+You can create complex filtering conditions by combining as many filtering conditions as needed.
```

Comment:
I mentioned the profile filters in the previous subsection and linked to the Glossary. Apart from the term definition, the Glossary article contains the link to the 'Profile filters' section of the documentation.